### PR TITLE
translated and updated nominate speaker page

### LINKED ISSUE #193: Translate secondary affiliation fields
Please provide Japanese translation for the following strings:

"Pronouns"
"Please type in your pronouns if yours are not listed"
"Secondary Role"
"Academic institutions, volunteer positions, or second jobs"
"(Secondary) Organization"

It's okay to provide these as a comment and then someone else can make the code change. See screenshot below for context, or visit: https://speakher.jp/nominate-speaker
(Google / Machine translation unacceptable for this task, please only work on this if you are fluent.)

### PR DESCRIPTION
**Pull request by Kimmym6a**
Resolves #193  
*Please link to the related issue, using the [Resolves] keyword only if the PR completes the entire issue*

**Proposed Changes**
I translated some English words and updated some not natural Japanese expressions.

*Description of changes in this PR*
Before
before

After
after

**Testing Plan**

*How do the reviewers QA this? Feel free to write unit tests or describe manual testing steps here*

### CODE REVIEW

File: web/src/i18n/locales/ja.json [hunk 1]
```diff
@@ -64,37 +64,46 @@
...
         },
         "secondaryAffiliation": {
+          "title": "経歴（その他）",
+          "organization": "所属先（その他）",
+          "hint": "学術研究、ボランティア、副業など、その他の経歴をお書きください。"
         },
         "languages": "言語",
         "linkedIn": "LinkedInのURL",
-        "nameEn": "お名前(英語\/ローマ字)",
-        "nameJa": "お名前（漢字\/かな）",
```
Comment (ann-kilzer) [inline web/src/i18n/locales/ja.json:84]: Some of us don't have Kanji names, like my name is キルザーアン, so we'd like to reference that kana are also acceptable.

**Review by ann-kilzer — COMMENTED** [review]
I think this is looking nice, just one request that we keep the kana suggestion.

Going to add @Ania-chan as a reviewer just because she helped with our original translations.


Thanks so much for helping improve our site!

File: web/src/i18n/locales/ja.json [hunk 2]
```diff
@@ -64,37 +64,46 @@
...
+        "nameEn": "お名前(英語)",
+        "nameJa": "お名前（漢字）",
```
Comment (ann-kilzer) [inline web/src/i18n/locales/ja.json:88]:
```suggestion
        "nameJa": "お名前（漢字\/かな）",```

Comment (Kimmym6a) [reply web/src/i18n/locales/ja.json:88]: name

Comment (Kimmym6a) [reply web/src/i18n/locales/ja.json:88]: I wrote down everything on the image.

Comment (ann-kilzer) [reply web/src/i18n/locales/ja.json:88]: This is interesting feedback and perhaps should be considered in a separate issue. Currently the design is focused on collecting one name for the Japanese version of the site, and one name for the English version of the site. Note how the names switch when changing the language in the toolbar.

For a Japanese user, they would enter Kanji / Kana for their name, and the pronunciation in Romaji
For a non-Japanese user, they could enter their name in Roman characters, then enter the Kana spelling.

We could consider alternate collection methods, but we'll want to keep it simple, and be inclusive of different names. (One of the reasons we don't ask for first or last name is to let people write their full name in the preferred order)

File: web/src/i18n/locales/ja.json [hunk 3]
```diff
@@ -64,37 +64,46 @@
...
     "nominateSpeaker": {
         "bio": {
-            "hint": "被推薦者の簡単な説明",
-            "label": "経歴"
+            "hint": "こ簡潔にご記入ください。",
+            "label": "ご経歴"
         },
-        "city": "都市",
-        "consent": "スピーカーの情報をSpeakHerデータベースに登録する許可をスピーカーから得ています。",
+        "city": "市区町村",
```
Comment (Ania-chan) [inline web/src/i18n/locales/ja.json:70]: @Kimmym6a I'm wondering do we need such a specific information here. I feel that the city only is enough, also for privacy

Comment (ann-kilzer) [reply web/src/i18n/locales/ja.json:70]: It's an optional field so if they don't want to put it, the form doesn't require it.

Comment (Ania-chan) [reply web/src/i18n/locales/ja.json:75]: @ann-kilzer I assume it's supposed to be used to search for the speakers by region so if that's the case I'd change it to `都道府県` to align it with this search (also not to ask users fro too detailed information about where they live): 
Screen Shot 2021-02-27 at 17 25 36

Comment (ann-kilzer) [reply web/src/i18n/locales/ja.json:75]:
```suggestion
        "city": "都道府県",
```

Comment (ann-kilzer) [reply web/src/i18n/locales/ja.json:75]: Ahh that makes sense. We should probably use "prefecture" for the English translation as well. I will update in a later PR.

**Review by ann-kilzer — APPROVED** [review]
LGTM after checking with @Ania-chan

File: web/src/i18n/locales/ja.json (hunk 3 above)
Comment (ann-kilzer) [reply web/src/i18n/locales/ja.json]: Nevermind, I am missing the context. We already collect prefecture.... now we ask for it twice

I will fix in a small PR

SpeakHer

**Pull request merged**